aries-site/src/examples/templates/wizard/components/StepContent.js

```diff
@@ -36,7 +65,9 @@ export const StepContent = () => {
             id={`${id}-form`}
             value={formValues}
             onChange={nextValue => setFormValues(nextValue)}
-            onSubmit={({ value }) => console.log(value)}
+            onSubmit={handleSubmit}
+            onValidate={onValidate}
+            validate="blur"
```

Comment:
My understanding is we want validation to happen when a user attempts to advance to the next step (Clicking "Next") to avoid throwing validation errors too often to the user. More inviting if we let them fill out the inputs and just display errors when they click "next" meaning they are trying to move on but we should tell them if something went wrong.

Reply:
Done. I switched it back to the default which is validation="submit". I instead make the onChange of the validated fields clear the error message at the very end of the form until the user hits the Next or Finish button again which seems less confusing. There's isn't another good way to figure out the valid state of the form from what I've seen. I thought grommet sets the aria-invalid property on elements but that doesn't seem to be true.